stumpy/__init__.py

```diff
@@ -13,6 +13,7 @@
 from .chains import atsc, allc  # noqa: F401
 from .floss import floss, fluss, _nnmark, _iac, _cac, _rea  # noqa: F401
 from .scrump import scrump  # noqa: F401
+from .search import search_k_motifs, search_occurrences  # noqa: F401
```

Comment:
Considering that `search_k_motifs` (and `search_k_discords`) is a top level API function, I would really want to keep the function name(s) short and, where possible, without underscores (`gpu_stump` was a necessary exception)

I still prefer `motifs()` and `discords()`. I think anybody looking for motifs/discords already know that they are "searching" and `k` is a default anyways and also in the function description so `search_k` is not needed (and users will adapt to it).

Additionally, should "search_occurrences" be a private function?

Reply:
Okay!

I don't think `search_occurrences` should be private: Suppose you know how a motif looks like, and you want to find it in a new time series, then you wold use this function.

Reply:
> I don't think search_occurrences should be private: Suppose you know how a motif looks like, and you want to find it in a new time series, then you wold use this function.

Could we rename this to `match()`? Only to simplify the function name